include/RAJA/operators.hxx

```diff
@@ -260,7 +260,7 @@ RAJA_HOST_DEVICE constexpr T max()
 template <typename Ret, typename Arg1 = Ret, typename Arg2 = Arg1>
 struct plus : public detail::binary_function<Arg1, Arg2, Ret>,
               detail::associative_tag {
-  RAJA_HOST_DEVICE Ret operator()(const Arg1& lhs, const Arg2& rhs)
+  RAJA_HOST_DEVICE Ret operator()(const Arg1& lhs, const Arg2& rhs) const
```

Comment:
Unfortunately, for some reason the Agency version seems to take all of these structs as `const`, so when it called `operator()` it dropped qualifiers. Given that there aren't any private data members in these structs, its a safe change to make all of them `const`.